packages/components/src/Loading/LoadingSpinner/LoadingSpinner.module.css

```diff
@@ -0,0 +1,28 @@
+.loadingSpinner {
+  --loading-spinner-size: var(--spacing-48);
+
+  display: flex;
+  width: var(--loading-spinner-size);
+  height: var(--loading-spinner-size);
+}
+
+.loadingSpinner.xs {
+  --loading-spinner-size: var(--spacing-16);
+}
+
+.loadingSpinner.sm {
+  --loading-spinner-size: var(--spacing-24);
+}
```

Comment:
We should try to keep specificity low where possible, so I'd remove the `.loadingSpinner` part. CSS modules will prevent name clashing.